tsconfig.json

```diff
@@ -1,14 +1,16 @@
 {
   "extends": "./.config/tsconfig.json",
-  "include": ["src", "types"],
```

Comment:
almost certainly we don't want to delete this file

Reply:
I believe the settings are set in the parent tsconfig, but it is harmless to have a more direct reference here, adding it back in.

Reply:
Restored to include it.